shared/js/background/atb.es6.js

```diff
@@ -82,22 +82,23 @@ var ATB = (() => {
         },
 
         calculateInitialVersions: () => {
-            let oneWeek = 604800000,
-                oneDay = 86400000,
-                oneHour = 3600000,
-                oneMinute = 60000,
-                estEpoch = 1456290000000,
-                localDate = new Date(),
-                localTime = localDate.getTime(),
-                utcTime = localTime + (localDate.getTimezoneOffset() * oneMinute),
-                est = new Date(utcTime + (oneHour * -5)),
-                dstStartDay = 13 - ((est.getFullYear() - 2016) % 6),
-                dstStopDay = 6 - ((est.getFullYear() - 2016) % 6),
-                isDST = (est.getMonth() > 2 || (est.getMonth() == 2 && est.getDate() >= dstStartDay)) && (est.getMonth() < 10 || (est.getMonth() == 10 && est.getDate() < dstStopDay)),
-                epoch = isDST ? estEpoch - oneHour : estEpoch,
-                timeSinceEpoch = new Date().getTime() - epoch,
-                majorVersion = Math.ceil(timeSinceEpoch / oneWeek),
-                minorVersion = Math.ceil(timeSinceEpoch % oneWeek / oneDay)
+            let oneWeek = 604800000
+            let oneDay = 86400000
+            let oneHour = 3600000
+            let oneMinute = 60000
+            let estEpoch = 1456290000000
+            let localDate = new Date()
+            let localTime = localDate.getTime()
+            let utcTime = localTime + (localDate.getTimezoneOffset() * oneMinute)
+            let est = new Date(utcTime + (oneHour * -5))
+            let dstStartDay = 13 - ((est.getFullYear() - 2016) % 6)
+            let dstStopDay = 6 - ((est.getFullYear() - 2016) % 6)
+            let isDST = (est.getMonth() > 2 || (est.getMonth() === 2 && est.getDate() >= dstStartDay)) && (est.getMonth() < 10 || (est.getMonth() === 10 && est.getDate() < dstStopDay))
+            let epoch = isDST ? estEpoch - oneHour : estEpoch
+            let timeSinceEpoch = new Date().getTime() - epoch
+            let majorVersion = Math.ceil(timeSinceEpoch / oneWeek)
+            let minorVersion = Math.ceil(timeSinceEpoch % oneWeek / oneDay)
```

Comment:
This felt a bit opinionated but ah well. :man_shrugging: Note I changed `isDST` to use strict equality for `getMonth`, too. This is safe, since the `getMonth` returns a number and not a string ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth))